pkg/datapath/linux/probes/probes.go

```diff
@@ -59,6 +58,17 @@ type kernelOption struct {
 	CanBeModule bool
 }
 
+type HelperProbe struct {
+	ProgramType ebpf.ProgramType
+	Helper      asm.BuiltinFunc
+}
+
+type HeaderProbes struct {
+	Maps               map[ebpf.MapType]bool
+	Helpers            map[HelperProbe]bool
+	HaveLargeInsnLimit bool
```

Comment:
This could already go into a `Misc` to prevent people from slapping on new bools here.